File: angular-app/src/app/model/order-dto.ts
export class OrderDto {
  uuid: string;
  title: string;
  scoreBoardNumber: number;
  state: OrderState;
  createdDate: Date;
  lastModifiedDate: Date;
}

export enum OrderState {
  IN_PROGRESS= "IN_PROGRESS",
  READY= "READY"
}

File: src/main/java/de/raychouni/order/adapter/in/web/dtos/OrderDto.java
package de.raychouni.order.adapter.in.web.dtos;

import java.util.Date;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class OrderDto {
    private String uuid;
    private String title;
    private int scoreBoardNumber;
    private String state;
    private Date createdDate;
    private Date lastModifiedDate;
}
